core/network/impl/protocols/protocol_req_collation.cpp

```diff
@@ -28,10 +28,10 @@ namespace kagome::network {
                                     host,
                                     make_protocols(kReqCollationProtocol,
                                                    genesis_hash,
-                                                   "polkadot"),
+                                                   chain_spec),
```

Comment:
[rust](https://github.com/paritytech/polkadot/blob/3fd990501423edca102bc473c2bb1c2ce9b93c23/node/network/protocol/src/request_response/mod.rs#L277-L286)
```suggestion
                                                  "polkadot"),
```

Reply:
as above